app/helpers/media_helper.rb

```diff
@@ -10,13 +10,20 @@ def media_thumbnail(media_item)
     end
   end
 
-  def media_title(media_item)
-    ext = File.extname(media_item.item.file.path)
-    file_name = truncate(File.basename(media_item.item.file.path, ext), length: 12)
+  def media_title(media_item, shorten: true)
```

Comment:
Note that I changed this to a named parameter, which makes it easier to read below.